requirements.txt

```diff
@@ -12,4 +12,9 @@ pytest==8.3.3
 httpcore==1.0.5
 httpx==0.27.2
 pytest-asyncio==0.24.0
-websockets==13.1
\ No newline at end of file
+websockets==13.1
+astroid==3.3.5 
+dill==0.3.9
+isort==5.13.2
+pylint==3.3.1
+tomlkit==0.13.2
```

Comment:
Good job setting up the CI workflow for both building and linting! The matrix and Python setup is clear and structured well.